rpm_spec/gui-agent.spec.in

```diff
@@ -71,7 +71,7 @@ BuildRequires:	mesa-libgbm-devel
 %endif
 BuildRequires:	pam-devel
 BuildRequires:	qubes-libvchan-devel
-BuildRequires:	qubes-gui-common-devel >= 4.1.0
+BuildRequires:	qubes-gui-common-devel >= 4.2.6
```

Comment:
Leave this to me after merge. This needs to be done only after the other version is committed (and preferably built, otherwise CI fails).

Reply:
Sure. Will do it in one hour. I am finalizing PR for issue 9029 (`Ability to hide (a) qube(s) from updates`) at the moment. And will come back to this shortly.

Reply:
I converted the status of this PR to draft (as well its related PRs). This needs much more work and testing.

Reply:
This is reverted to previous situation